gpii/node_modules/flowManager/test/CaptureTests.js

```diff
@@ -47,22 +47,31 @@ gpii.tests.flowManager.capture.platformReporter = function () {
     };
 };
 
+fluid.defaults("gpii.tests.flowManager.capture.platformReporter", {
+    gradeNames: "fluid.function"
+});
+
 gpii.tests.flowManager.capture.checkForInstalledMags = function (payload) {
     jqUnit.assertEquals("There should be 2 solutions installed", 2, Object.keys(payload).length);
     jqUnit.assertEquals("Check for Fake Mag 1", "Fake Magnifier 1", payload.fakemag1.name);
     jqUnit.assertEquals("Check for Fake Mag 2", "Fake Magnifier 2 - fully featured", payload.fakemag2.name);
 };
 
-gpii.tests.flowManager.capture.adjustSolutions = function (solutionsRegistryDataSource) {
+gpii.tests.flowManager.capture.adjustSolutions = function (solutionsRegistryDataSource, revision, platformReporter) {
+    var platformId = fluid.invokeGlobalFunction(platformReporter).id;
```

Comment:
This test has defined the use of "darwin" platform at line 46. I think it's unnecessary to have line 50-51 and fetch this known id from `gpii.tests.flowManager.capture.platformReporter`. To simplify, you could define `gpii.tests.flowManager.capture.platformId = "darwin"` at the beginning of the test file and use this variable whenever needed.

Reply:
Okay